### ISSUE TITLE
Zip Attachment on S/MIME signed messages

### ISSUE THREAD

**Issue by somjue** [MailScanner/v5#637]
If Zip Attachment = yes on a S/MIME signed message (Content-Type: multipart/signed;), the MessageAttachments.zip is attached to the wrong mime-part, so some mail clients (Thunderbird) won't show the attachment. Attached two sample e-mails one with current result and one with correct headers.

Maybe it's also to discuss, to disable Zip Attachment because signature will be invalid after attachment zip. 

Regards,
Juerg

[current-wrong.txt](https://github.com/MailScanner/v5/files/10290718/current-wrong.txt)
[correct.txt](https://github.com/MailScanner/v5/files/10290719/correct.txt)

**Comment by shawniverson** [comment]
I agree that Zip Attachment doesn't make much sense with an S/MIME message.  I'll try to wrap my head around why the mine boundary is missing and work on this.
